src/coreclr/vm/comutilnative.cpp

```diff
@@ -479,6 +479,54 @@ extern "C" void QCALLTYPE ExceptionNative_ThrowEntryPointNotFoundException(
     END_QCALL;
 }
 
+extern "C" void QCALLTYPE ExceptionNative_ThrowMethodAccessException(CORINFO_METHOD_HANDLE caller, CORINFO_METHOD_HANDLE callee)
```

Comment:
Nit: `CORINFO_METHOD_HANDLE` and similar types should be only used on the JIT/EE interface (ie compile-time). A more appropriate type for the JIT helpers (ie runtime-time) would be `MethodDesc*`, `MethodTable*`, `EnregisteredTypeHandle`, etc.